src/node/desktop/scripts/rename-dmg.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Nit: This form is preferred so that the version of bash on `$PATH` is used.